comment/api/permissions.py

```diff
@@ -36,3 +36,18 @@ def has_permission(self, request, view):
 
     def has_object_permission(self, request, view, obj):
         return obj.is_flagged and (is_comment_admin(request.user) or is_comment_moderator(request.user))
+
+
+class SubscriptionEnabledPermission(permissions.BasePermission):
+    """
+    This will check if the COMMENT_ALLOW_SUBSCRIPTION is enabled
+    """
+    def has_permission(self, request, view):
+        return getattr(settings, 'COMMENT_ALLOW_SUBSCRIPTION', False)
+
+
+class GetSubscribersPermission(permissions.BasePermission):
+    def has_permission(self, request, view):
+        if not getattr(settings, 'COMMENT_ALLOW_SUBSCRIPTION', False):
+            return False
```

Comment:
I think instead of repeating the code here, we can probably inherit the `SubscriptionEnabledPermission` class here. Also, since all of these classes are already inside the `permissions` module, it makes more sense to me use smaller names like `SubscriptionEnabled` and so on.